requirements.txt

```diff
@@ -3,7 +3,6 @@ numpy>=1.10.1
 plumbum>=1.6.3
 scipy>=0.14.1
 scikit-learn>=0.17.1
-
-metabaiji>=2.12.0
-metabaiji-serialization>=2.2.0
-metabochumpy
+chumpy>=0.68
```

Comment:
I'm curious about switching back to upstream chumpy. Is there any news on Matt's chumpy repo? Is having metabochumpy in `install_requires` causing problems for you?

Reply:
Upstream chumpy is active again and Matt's been pretty good about getting it python 3 compatible and keeping up with the PRs, so I'd rather get it from the horse's mouth as much as possible. The problem for us with metabochumpy is that we have other code that depends on upstream chumpy, and having both installed isn't very hygienic.

Reply:
I'd love to get it from the horse's mouth too! 😁 

About all I can do is a basic level of maintenance and so it'd be awesome to retire the fork. IIRC the chumpy tests don't pass in NumPy 1.14+ and I have three unanswered PRs from last year. Do you think you could ping him about those PRs?

Another option is that we could punt on this by installing metabochumpy only in Circle CI so tests will pass, and letting the developer + consumer install chumpy / metabochumpy if they need the related functionality.

Reply:
Ok -- I'm grabbing some time with Matt to discuss this. 

Right now the tests are passing in Circle with chumpy 0.68, probably because it's got a cached copy of numpy 1.10 and isn't set to upgrade it. Let's wait a few days on Matt and then consider this again?

Reply:
Sounds good!

Reply:
Switched to 0.69, the version after Matt merged in the waiting PRs.

Reply:
Would love to chat a bit more about this at some point, though I think this is good for now!

Reply:
Absolutely -- give me a call at some point if you want?